cli/pkg/backend/aws/k8s_compute_backend.go

```diff
@@ -281,3 +300,71 @@ func (k8s *K8SComputeBackend) Shell(ctx context.Context, stackName string, servi
 	streamOptions.TerminalSizeQueue = t.MonitorSize(t.GetSize())
 	return t.Safe(func() error { return exec.Stream(streamOptions) })
 }
+
+func (k8s *K8SComputeBackend) GetEvents(ctx context.Context, stackName string, services []string) error {
+	if len(services) == 0 {
+		return nil
+	}
+
+	for _, serviceName := range services {
+		pods, err := k8s.getPods(ctx, stackName, serviceName)
+		if err != nil {
+			return errors.Wrap(err, "unable to retrieve a list of pods")
+		}
+
+		if err != nil {
+			return errors.Wrap(err, "unable to retrieve a list of pods")
+		}
```

Comment:
error handling is duplicated
```suggestion
		pods, err := k8s.getPods(ctx, stackName, serviceName)
		if err != nil {
			return errors.Wrap(err, "unable to retrieve a list of pods")
		}
```